### ISSUE TITLE
[EPUB 3.3] [explanation] Linked resources

### ISSUE THREAD

**Issue by gregoriopellegrino** [w3c/epub-specs#1730]
As a non-english speaker I find difficulty in understanding the sentence.

> Linked resources are not Publication Resources. EPUB Creators MUST NOT list them in the manifest. EPUB Creators MUST use linked resources that are Core Media Type Resources to embed them in Publication Resources listed in the manifest (e.g., an EPUB Content Document could contain a metadata record serialized as RDFa [RDFA-CORE] or JSON-LD [JSON-LD11]).
> https://www.w3.org/TR/epub-33/#sec-link-elem

Perhaps a rephrase can be thought of?

**Comment by mattgarrish** [comment]
Am I right that it's this part you find confusing:

> EPUB Creators MUST use linked resources that are Core Media Type Resources to embed them in Publication Resources listed in the manifest (e.g., an EPUB Content Document could contain a metadata record serialized as RDFa [RDFA-CORE] or JSON-LD [JSON-LD11]).

This is hard to parse in English, too. 😄 

The point is that if you embed a linked resource, it must be a core media type. The exemption no longer applies.

Easier said than specified, though. But what about something more like this:

> Linked Resources are Publication Resources only when they are:
> 
> - referenced from the spine; or
> - included or embedded in an EPUB Content Document (e.g., a metadata record serialized as RDFa [RDFA-CORE] or JSON-LD [JSON-LD11] in an [HTML] `script` element).
> 
> In these cases, EPUB Creators MUST ensure the linked resources meet Core Media Type requirements. In all other cases the linked resources are not Publication Resources (i.e., are not subject to Core Media Type requirements) and EPUB Creators MUST NOT list them in the manifest.

**Comment by iherman** [comment]
I like the alternative from @mattgarrish. As part of the clarification, we could add a reference to metadata records as example. Ie, it would say something like:

> In these cases, EPUB Creators MUST ensure the linked resources meet Core Media Type requirements. In all other cases (e.g., when linking to ONIX or XMP records) the linked resources are not Publication Resources (i.e., are not subject to Core Media Type requirements) and EPUB Creators MUST NOT list them in the manifest.

**Comment by gregoriopellegrino** [comment]
Fine for me, thanks
